### PR TITLE
Smooth modal integration

### PR DESCRIPTION
**Pull request by GiselleNessi**
Show smooth modal

Please see issue #1808

Screenshot 2024-02-26 at 5 36 22 PM

Screenshot 2024-02-27 at 1 08 48 PM

### CODE REVIEW

**Review by Marketen — COMMENTED** [review]
Everything related to Smooth looks correct

**Comment by Marketen** [comment]
Screenshot_20240227_161156
Screenshot_20240227_161550

when I switch my dappmanager to this branch, some popup texts appear with wrong sizes, unsure if its related to these changes

File: packages/admin-ui/src/components/smooth.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
+.title {
```
Comment (Marketen) [inline packages/admin-ui/src/components/smooth.scss:1]: i would change this identifier to something more "smooth related" not that generic

File: packages/admin-ui/src/components/smooth.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    font-weight: bold;
+    font-size: 2rem; 
+    transition: color 0.15s;
+    color: #000;
+    margin-top: 20px;
+    text-align: center !important;
+  }
+
+.description {
```
Comment (Marketen) [inline packages/admin-ui/src/components/smooth.scss:10]: this one too

**Review by Marketen — CHANGES_REQUESTED** [review]

**Review by Marketen — APPROVED** [review]
LGTM

**Review by pablomendezroyo — APPROVED** [review]

**Pull request merged**